lib/msf/core/auxiliary/report.rb

```diff
@@ -401,37 +406,37 @@ def store_loot(ltype, ctype, host, data, filename=nil, info=nil, service=nil)
     host = Msf::Util::Host.normalize_host(host)
 
     ws = (db ? myworkspace.name[0,16] : 'default')
-    name =
-      Time.now.strftime("%Y%m%d%H%M%S") + "_" + ws + "_" +
-      (host || 'unknown') + '_' + ltype[0,16] + '_' +
-      Rex::Text.rand_text_numeric(6) + '.' + ext
-
+    name =Time.now.strftime("%H:%M:%S")+"_"+ws + "_" +
```

Comment:
Please pay attention to the alignment of code, which could make it more readable and lucid. 
I guess you change it by accident, correct it please.